docs/UserGuide.md

```diff
@@ -116,17 +116,17 @@ This detailed guide will walk you through how to make the most of **EduManage**
 |   5   |       Level       |                   `l/LEVEL`                    | Should be in the format `YEAR TRACK`, where:<ul><li>either:<ul><li>`YEAR` is one of: [S1, S2, S3, S4] and</li><li>`TRACK` is one of: [EXPRESS, NA, NT, IP].</li></ul><li>or:<ul><li>`YEAR` is S5 and</li><li>`TRACK` is NA.</ul></ul>                                                                                                                                                                                                                                                                                                                                                                                                                       |
 |   6   |       Name        | `n/NAME` or `update NAME` or `find n/KEYWORDS` | Should only contain alphanumeric characters and spaces, and should not be blank. For commands other than `add n/NAME`, `update [n/NAME]` and `find n/KEYWORDS`, the provided name must match an existing student's name in EduManage. Refer to [Section 5 - Known Issues](#5-known-issues) for guidance on handling valid names that contain special characters.                                                                                                                                                                                                                                                                                            |
-|   7   |       Note        |                   `nt/NOTE`                    | Can take any values except prefixes already reserved, and can be blank.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+|   7   |       Note        |                   `nt/NOTE`                    | Can take any values, and can be blank. Note that using certain prefixes may cause an error.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Same as above, change to "Note that using reserved prefixes within the note may cause an error." for clarity?